identity/resource_token.go

```diff
@@ -50,10 +50,16 @@ type TokensAPI struct {
 
 // Create creates a api token given a expiration duration and a comment
 func (a TokensAPI) Create(tokenLifetime time.Duration, comment string) (r TokenResponse, err error) {
-	err = a.client.Post(a.context, "/token/create", TokenRequest{
-		LifetimeSeconds: int32(tokenLifetime.Seconds()),
-		Comment:         comment,
-	}, &r)
+	request := TokenRequest{}
+	seconds := int32(tokenLifetime.Seconds())
+	if seconds > 0 {
+		request.LifetimeSeconds = seconds
+	}
```

Comment:
can you introduce a constant in the top of the file, `DefaultTokenLifetimeSeconds`, and make it to 3600 and reference it here?...

Reply:
I prefer not to.  How would you be able to create tokens without an expiration then? Seconds == 0 should mean that the token doesn't expire?

Reply:
I think it will fail, no?

Reply:
No, I posted the API docs in #494, you should be able to create tokens without comments and without expiration

Reply:
@sdebruyn Oh, okay. then i'll ask for one last thing - can you add `TestResourceTokenCreate_NoExpiration`, that would verify it on the resource level. I just suspect we'll have the same issue as with single node clusters. When i'll see that one new unit test (you've added integration test) with creating token with no arguments - i'll merge :)